docs/introduction/Features.md

```diff
@@ -40,20 +40,22 @@ StarRocks can load data within seconds for near-real-time analytics. StarRocks'
 
 ![Realtime](../assets/1.1-6-realtime.png)
 
-StarRocks' storage engine uses the Delete-and-insert (Merge on Read) pattern, which allows for efficient Append and UPSERT operations. The storage engine can quickly filter data using primary key indexes,  eliminating the need for Sort and Merge operations at data reading. The engine can also make full use of secondary indexes. It delivers ultimate query performance even on huge amounts of data updates.
+StarRocks' storage engine uses the Delete-and-insert pattern, which allows for efficient Partial Update and Upsert operations. The storage engine can quickly filter data using primary key indexes,  eliminating the need for Sort and Merge operations at data reading. The engine can also make full use of secondary indexes. It delivers fast and predictable query performance even on huge amounts of concurrent data updates.
```

Comment:
Huge amounts of `concurrent` data updates is not recommended, no more than 1 tps per table for primary key storage engine.

Reply:
> Huge amounts of `concurrent` data updates is not recommended, no more than 1 tps per table for primary key storage engine.

Got it